data/python/jepeval.py

```diff
@@ -81,4 +81,23 @@ def _jepeval(line):
 
         traceback.print_exc()
 
+        def log_env_details():
+            import platform
+
+            import jep
+            from java.lang import System
+
+            print(
+                "Python={python_version}, Arch={arch}, OS={os}, Ghidra={ghidra_version}, Java={java_version}, Ghidrathon=4.0.0, Jep={jep_version}".format(
```

Comment:
If I am not wrong, Ghidrathon doesn't expose `ghidrathon.__version__`, thus the hardcoded `4.0.0` version number. Is there a dynamic way to do this?

The following showcases the execution of `log_env_details` when an exception is raised in Ghidrathon:

![grafik](https://github.com/mandiant/Ghidrathon/assets/31004043/5a22a16f-bef3-4796-af1e-dab84f985fb4)

Reply:
We don't expose Ghidrathon's version from Java but this would be a reasonable thing to do. We could add a `static` member to https://github.com/mandiant/Ghidrathon/blob/main/src/main/java/ghidrathon/GhidrathonPlugin.java that contains a version string? This should give us access in both Java and Python. Thoughts?

Reply:
I agree with this approach. Please refer to my comment on [c1a8400](https://github.com/mandiant/Ghidrathon/pull/101/commits/c1a84003b1d3b69a6d45ff8c91c442df7dd5128d)  for some context.